client/js/app/utils/ExplorerUtils.js

```diff
@@ -265,6 +265,10 @@ module.exports = {
       delete attrs['steps'];
     }
 
+    if (attrs.group_by) {
```

Comment:
Ok cool, I'll make that change now and write another test to cover the array and non-array use cases.